### DISCUSSION TITLE
Transferring Github Student Developer MKit from one GitHub account to another

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35754] [GitHub Education]
Hi! 
I have registered for GitHub Student Developer Kit and my application got approved. 
Tl;dr, I need to switch from my GitHub ID to another GitHub ID. Is there a way by which I can transfer the Student Developer Kit (GitHub Pro) from the old ID to the new ID?

**Comment by samus-aran** [comment]
Hi @VoidCupboard 👋 

You're able to change [your username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username) on the verified account, as well as making changes to [your emails (add/removing)](url). However you **can not** transfer your Global Campus benefits to another account. We'll take this opportunity to mention that having multiple free GitHub accounts would be a violation of [GitHub's Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements).

**Comment by ghost** [comment]
Hi @samus-aran ^-^,
I am actually aware of the fact that I am able to change both my username as well as my email. 
I understand the violation, thanks for the response!
